config/environments/development.rb

```diff
@@ -52,10 +52,13 @@
   # config.i18n.raise_on_missing_translations = true
 
   # Annotate rendered view with file names.
-  # config.action_view.annotate_rendered_view_with_filenames = true
+  config.action_view.annotate_rendered_view_with_filenames = true
```

Comment:
Not a blocker, but would be nice to call out why we're enabling this

Reply:
It is a change suggested by app:update.

As it only applies to the development environment, it's safe to apply with no impact on tests or production.

It helps in debugging. Here is the description of what it does: https://www.bigbinary.com/blog/rails-6-1-adds-annotate_rendered_view_with_filenames-to-annotate-html-output

What is your opinion on that?

Reply:
I think 'It helps in debugging.' is enough. Just so people looking back can understand why we decided to uncomment it.

Reply:
Also I think this conversation is probably enough documentation too! So I think all good to be merged! 💪🏻